From foo@baz Mon Jul 13 17:07:49 PDT 2009
Date: Mon, 13 Jul 2009 17:07:49 -0700
From: Greg Kroah-Hartman <gregkh@suse.de>
Date: Mon, 13 Jul 2009 16:04:02 -0700
Subject: Staging: hv: add the Hyper-V virtual bus to the build

From: Greg Kroah-Hartman <gregkh@suse.de>

Add the Hyper-V virtual bus to the kernel build system.

Cc: Hank Janssen <hjanssen@microsoft.com>
Cc: Haiyang Zhang <haiyangz@microsoft.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/Kconfig     |    2 ++
 drivers/staging/Makefile    |    1 +
 drivers/staging/hv/Kconfig  |    8 ++++++++
 drivers/staging/hv/Makefile |    3 +++
 4 files changed, 14 insertions(+)

--- a/drivers/staging/Kconfig
+++ b/drivers/staging/Kconfig
@@ -57,4 +57,6 @@ source "drivers/staging/benet/Kconfig"
 
 source "drivers/staging/rtl8187se/Kconfig"
 
+source "drivers/staging/hv/Kconfig"
+
 endif # STAGING
--- a/drivers/staging/Makefile
+++ b/drivers/staging/Makefile
@@ -20,3 +20,4 @@ obj-$(CONFIG_RT2870)		+= rt2870/
 obj-$(CONFIG_RT3070)		+= rt3070/
 obj-$(CONFIG_BENET)		+= benet/
 obj-$(CONFIG_RTL8187SE)		+= rtl8187se/
+obj-$(CONFIG_HYPERV)		+= hv/
--- /dev/null
+++ b/drivers/staging/hv/Kconfig
@@ -0,0 +1,8 @@
+config HYPERV
+	tristate "Microsoft Hyper-V client drivers"
+	depends on X86 && !XEN
+	default n
+	help
+	  Select this option to run Linux as a Hyper-V client operating
+	  system.
+
--- /dev/null
+++ b/drivers/staging/hv/Makefile
@@ -0,0 +1,3 @@
+obj-$(CONFIG_HYPERV)		+= hv_vmbus.o
+
+hv_vmbus-objs := vmbus_drv.o osd.o Sources.o
